features/FEATURE_BLE/ble/GattClient.h

```diff
@@ -115,16 +115,17 @@ class GattClient {
          * It is used to request the server to write the value of an attribute.
          * The server does not acknowledge the status of the operation.
          */
-        GATT_OP_WRITE_CMD = 0x02,
-
+        GATT_OP_WRITE_CMD = 0x02
+#if BLE_FEATURE_SIGNING
         /**
          * Signed Write command.
          *
          * It is used to request the server to write the value of an attribute
          * using a signed packet. The server does not acknowledge the status
          * of the operation.
          */
-        GATT_OP_SIGNED_WRITE_CMD = 0x03
+        , GATT_OP_SIGNED_WRITE_CMD = 0x03
```

Comment:
`,` on the wrong line.

Reply:
awkward `,` placement needed because options remove items from lists

Reply:
Yeah I realised that with the other. I though I had cancelled that comment. Sorry.

Reply:
actually, this one is safe to leave at the end of the line (trailing commas OK in enums)